### ISSUE TITLE
[Supporting Platform - Bot Framework Web Chat - Card Breakfast]: In 320*256 pixel mode 'White-labelled', 'Copilot' and 'Fluent UI' controls are not visible.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#5364]
### Is it an issue related to Adaptive Cards?

No.

### User Experience: 
When users with low vision resize the page to the viewport of 320*256 pixel, it will be difficult for them to understand the content if the contents or controls on the page appear overlapped or truncated as they are the source of visual information on the page.
### Pre-requisite:
Achieve viewport 320 X 256 by adjusting the browser. (Open the developer's tool, resize, and zoom the browser to achieve viewport 320 X 256).
### Note:
User credentials should not be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://compulim.github.io/webchat-loader

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Breakfast word in type your message text box.
4. Navigate to the results.
5. Verify whether the controls are visible in 320*256 pixel mode or not.

### What do you expect?

In 320*256 pixel mode 'White-labelled', 'Copilot' and 'Fluent UI' controls should be visible clearly.

### What actually happened?

In 320*256 pixel mode 'White-labelled', 'Copilot' and 'Fluent UI' controls are not visible.
**Observation:** In normal mode the controls are clearly visible.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/67138d6c-5467-491e-9bdf-b0c7120cf619
![Image](https://github.com/user-attachments/assets/627ccec3-cc01-4764-a88e-8c340131a433)
![Image](https://github.com/user-attachments/assets/69ac9547-dae8-452b-91b8-9c9b92d250ef)

### MAS reference

MAS 1.4.10 – Reflow

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev1;#A11yWcag.2.1;#HCL;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#MAS1.4.10;#Win11-Edge(Chromium);

**Comment by compulim** [comment]
The radio button is not part of Web Chat. It's part of the sandbox app that we use to test the different functionality of Web Chat.

They should not be checked against accessibility.

**Comment by Kirank915** [comment]
As per the above comment control is not part of the web chat and closed the bug.
